src/node_file.cc

```diff
@@ -1496,12 +1377,20 @@ static void Mkdtemp(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
-void FSInitialize(const FunctionCallbackInfo<Value>& args) {
-  Local<Function> stats_constructor = args[0].As<Function>();
-  CHECK(stats_constructor->IsFunction());
-
+void GetStatValues(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
-  env->set_fs_stats_constructor_function(stats_constructor);
+  double* fields = env->fs_stats_field_array();
+  if (fields == nullptr) {
+    // stat fields contains twice the number of entries because `fs.StatWatcher`
+    // needs room to store data for *two* `fs.Stats` instances.
+    fields = new double[2 * 14];
+    env->set_fs_stats_field_array(fields);
+  }
+  Local<ArrayBuffer> ab = ArrayBuffer::New(env->isolate(),
```

Comment:
That's the [default mode](https://v8docs.nodesource.com/node-7.4/d5/d6e/classv8_1_1_array_buffer.html#a98c19b0bf91cc3ea22371fbab5707e25). Also the description says:

> Create a new **ArrayBuffer** over an existing memory block. The created array buffer is by default immediately in externalized state. The memory block will not be reclaimed when a created **ArrayBuffer** is garbage-collected.